sbt/sbt-openapi-codegen/src/main/scala/codegen/openapi/models/OpenapiModels.scala

```diff
@@ -0,0 +1,131 @@
+package codegen.openapi.models
+
+// https://swagger.io/specification/
+object OpenapiModels {
+
+  case class OpenapiDocument(
```

Comment:
can't we reuse the model that's already in tapir?

Reply:
I'm not sure. I watched the generator code, and I had the feeling that generating and parsing are totally different.

Now I checked the models again, and I would say a "maybe" :D My main problem was generating from the format that we has in the models are easy, but parsing and understanding it a bit harder. So I started with a "minimal" scope.

Reply:
Makes sense, and maybe the two models are distinct as they server different purposes. But maybe it's possible to use a single one and not duplicate the code, which would be nice :)